# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../boomerang/gcc41.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- boomerang/include/proc.h.vanilla	2006-07-24 16:16:38.000000000 +0000
+++ boomerang/include/proc.h	2006-07-24 16:16:48.000000000 +0000
@@ -165,7 +165,7 @@
 		/**
 		 * Get a list of types to cast a given list of actual parameters to
 		 */
-		std::list<Type>* Proc::getParamTypeList(const std::list<Exp*>& actuals);
+		std::list<Type>* getParamTypeList(const std::list<Exp*>& actuals);
 
 		/**
 		 * Return true if this is a library proc
